File: VB/Solution5/Solution5.Module.Win/WindowController1.vb
Imports Microsoft.VisualBasic
Imports System
Imports DevExpress.ExpressApp
Imports DevExpress.Persistent.Base

Namespace Solution5.Module.Win
	Public Class WindowController1
		Inherits WindowController
		Public Sub New()
			Dim action As New CheckableSimpleAction(Me, "TestAction", PredefinedCategory.RecordEdit)
		End Sub
	End Class


End Namespace

File: CS/Solution5.Module.Win/WindowController1.cs
using DevExpress.ExpressApp;
using DevExpress.Persistent.Base;

namespace Solution5.Module.Win
{
    public class WindowController1 : WindowController
    {
        public WindowController1()
        {
            CheckableSimpleAction action = new CheckableSimpleAction(this, "TestAction", PredefinedCategory.RecordEdit);
        }
    }
  
   
}
